src/routes/console/project-[project]/messaging/wizard/emailFormList.svelte

```diff
@@ -1,5 +1,27 @@
+<script context="module" lang="ts">
+    export async function createEmailMessage(params: EmailMessageParams) {
+        const response = await sdk.forProject.client.call(
+            'POST',
+            new URL(sdk.forProject.client.config.endpoint + '/messaging/messages/email'),
+            {
+                'X-Appwrite-Project': sdk.forProject.client.config.project,
+                'content-type': 'application/json',
+                'X-Appwrite-Mode': 'admin'
+            },
+            params
+        );
+
+        return response.json();
+    }
```

Comment:
I don't think we need this anymore since we have the createEmail() method in the SDK.